tools/run_tests/xds_k8s_test_driver/framework/xds_k8s_testcase.py

```diff
@@ -196,7 +196,7 @@ def _cleanup(self):
                                    force_namespace=self.force_cleanup)
 
     @staticmethod
-    def _random_resource_suffix() -> str:
+    def random_resource_suffix() -> str:
```

Comment:
I think at this point it'll be great to move it to `framework.helpers.rand`

Reply:
Cool, done.

Reply:
Thanks!